RHEL/7/input/profiles/stig-rhel7-disa.xml

```diff
@@ -243,7 +251,7 @@ This profile aligns with the DISA STIG for Red Hat Enterprise Linux V1R1.
 <!-- TO DO -->
 
 <!-- SRG-OS-000480-GPOS-00227, SV-86637r1_rule, RHEL-07-020610 -->
-<!-- TO DO -->
+<select idref="accounts_have_homedir_login_defs" selected="true" />
```

Comment:
I have noticed a weird issue. Although this was added here, in the built XCCDF there isn't any '<select>' element in any profile with idref="accounts_have_homedir_login_defs"